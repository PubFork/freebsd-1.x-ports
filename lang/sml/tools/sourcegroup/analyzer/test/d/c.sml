structure A = struct val x = B.y end
structure H = struct val y = A.x end
