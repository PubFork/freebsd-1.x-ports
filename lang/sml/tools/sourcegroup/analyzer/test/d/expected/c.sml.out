source sml "d/c.sml"
  import structure B 
  export structure A H ;

